geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/data/KafkaDataStore.scala

```diff
@@ -235,7 +272,7 @@ object KafkaDataStore extends LazyLogging {
 
   val MetadataPath = "metadata"
 
-  val LoadIntervalProperty: SystemProperty = SystemProperty("geomesa.kafka.load.interval", "100ms")
+  val LoadIntervalProperty: SystemProperty = SystemProperty("geomesa.kafka.load.interval", "1s")
```

Comment:
Any particular reason to change this?  (I'm fine either way; just asking.)

Reply:
this prop controls the amount of time the consumer will wait for a message when calling poll. i was reading some things about how if the data isn't processed within the poll interval of the consumer, then kafka might consider the consumer dead. i'm not really sure if this matters or not, but i don't think there's any downside to increasing it, b/c we just loop and poll again after the timeout.